docs/src/main/sphinx/client/cli.rst

```diff
@@ -206,6 +206,12 @@ mode:
       operating system username. You can override the default username,
       if your cluster uses a different username or authentication mechanism.
 
+Most of the options can also be set as parameters in the URL. This means
+a JDBC URL can be used in the CLI after removing the ``jdbc:`` prefix.
+However, the same parameter may not be specified using both methods.
+See :doc:`the JDBC driver parameter reference </client/jdbc>`
```

Comment:
We should have at least one example of a jdbc url and show it here explicitly,

Reply:
Added, PTAL.